test/e2e/network/netpol/k8s_util.go

```diff
@@ -99,8 +103,17 @@ func (k *Scenario) InitializeCluster(model *Model) error {
 	return nil
 }
 
-// GetPodFromCache returns a pod with the matching namespace and name
-func (k *Scenario) GetPodFromCache(ns string, name string) (*v1.Pod, error) {
+// clearPodCache clears the kube pod cache
```

Comment:
Can we make this a sentence and explain when its 'meant' to be called?  Antonio was asking about it during the review

Reply:
dug into this and it looks like the cache isn't really used so I just took it out ... can roll that back if I missed something, but I think we're better off without it!